test/interval_tests/trig.jl

```diff
@@ -91,15 +91,15 @@ end
     @test acos(@biginterval(3, 4)) ⊆ acos(@interval(3, 4))
 
     @test atan(@interval(-1,1)) ==
-        Interval(-pi_interval(Float64).hi/4, pi_interval(Float64).hi/4)
+        Interval(-Interval{Float64}(π).hi/4, Interval{Float64}(π).hi/4)
     @test atan(@interval(0)) == Interval(0.0, 0.0)
     @test atan(@biginterval(-1, 1)) ⊆ atan(@interval(-1, 1))
 end
 
 @testset "atan" begin
     @test atan(∅, entireinterval()) == ∅
     @test atan(entireinterval(), ∅) == ∅
-    @test atan(@interval(0.0, 1.0), @biginterval(0.0)) == @biginterval(pi/2)
+    @test_broken atan(@interval(0.0, 1.0), @biginterval(0.0)) == @biginterval(pi/2)
```

Comment:
Any clue why this test is broken now?

Reply:
It is because float and bigfloats intervals are mixed, so the `Base.Math` implementation is used:

```julia
julia> @which atan(@interval(0, 1), @biginterval(0))
atan(y::Real, x::Real) in Base.Math at math.jl:639
```

The fact that the test wasn't previously broken is probably a coincidence.

Reply:
It was only part of the problem apparently, as the test is still broken. I don't understand why though, because the test passes if I run only a this single test file.